planet.js

```diff
@@ -777,23 +777,34 @@ planet.connect = async (rn, {online = true} = {}) => {
 window.addEventListener('load', () => {
   const button = document.getElementById('connectButton');
   if (button) {
-    document.getElementById('connectButton').addEventListener('click', (e) => {
+    document.getElementById('connectButton').addEventListener('click', async (e) => {
```

Comment:
This could probably be moved outside of window load.

Reply:
probs fine

Reply:
No it's not.

Reply:
Is there a reason for this timing?